common/beerocks/bwl/dwpal/mon_wlan_hal_dwpal.cpp

```diff
@@ -10,13 +10,17 @@
 
 #include <bcl/beerocks_utils.h>
 #include <bcl/network/network_utils.h>
+#include <bcl/son/son_wireless_utils.h>
 
 #include <easylogging++.h>
+#include <net/if.h>
 
 #include <cmath>
+#include <functional>
 
 extern "C" {
 #include <dwpal.h>
+#include <slibc/stdio.h>
```

Comment:
This has to be fixed - it will either not work for UGW since in UGW we don't have this - which makes me wonder - have you tested these compiles in UGW? This is a blocker for merging this PR.

Reply:
removed.
already included in base under USE_LIBSAFEC